airflow/bin/cli.py

```diff
@@ -1019,8 +1019,7 @@ def kill_proc(dummy_signum, dummy_frame):
         signal.signal(signal.SIGINT, kill_proc)
         signal.signal(signal.SIGTERM, kill_proc)
 
-        while True:
```

Comment:
wow, not even a sleep?! these people are monsters

Reply:
These people == me in this case :(